kernel/src/process.rs

```diff
@@ -47,11 +30,12 @@ extern "C" {
 /// number of processes are created, with process structures placed in the
 /// provided array. How process faults are handled by the kernel is also
 /// selected.
-pub unsafe fn load_processes(
+pub unsafe fn load_processes<S: SyscallInterface>(
     kernel: &'static Kernel,
+    syscall: &'static S,
```

Comment:
Does this actually need to be a reference? If not, we can avoid the annoying `static_init!` of a zero-sized value in all the board configurations.

Reply:
Take a look at the commit I just added. It just removes the static_init!() call with no other changes.